src/components/Home/Header/index.tsx

```diff
@@ -369,24 +407,24 @@ const Header: React.FC = () => {
         </div>
       </div>
       <ul className={styles.header__logos}>
-        {logosData.map(({ widthSm, widthMd, widthLg, src, alt }, i) => (
-          <li key={i}>
-            <img
-              alt={alt}
-              src={src}
-              className={styles.header__logo}
-              width={widthMd}
-              height={35}
-              style={
-                {
-                  '--width-sm': `${widthSm}px`,
-                  '--width-md': `${widthMd}px`,
-                  '--width-lg': `${widthLg}px`
-                } as React.CSSProperties
-              }
-            />
-          </li>
-        ))}
+        {logosData.map((logoDetails, i) =>
+          logoDetails?.link ? (
+            <Link
+              href={logoDetails.link}
+              key={i}
+              className={cn(
+                styles.header__list,
+                'hover:border-purple-800 active:bg-gray-200 transition-colors'
+              )}
+            >
+              <LogoImage {...logoDetails} />
+            </Link>
+          ) : (
+            <li key={i} className={cn(styles.header__list)}>
+              <LogoImage {...logoDetails} />
+            </li>
+          )
+        )}
```

Comment:
Theoretically if any `logoDetails` were undefined, this would blow up. That said, that's probably not going to happen.